docs/workflow-actor.md

```diff
@@ -0,0 +1,28 @@
+# Workflow Actor
+
+> v3.6.5 and after
```

Comment:
Let's align on this,
I feel like it's a feature, should be on 3.7 but it's small enough and no harm to include in the next patch